packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -2306,24 +2306,39 @@ export const handlers = [
   }),
   http.get('*/monitor/services', () => {
     const response = {
-      data: [{ service_type: 'linode' }],
+      data: [
+        {
+          label: 'Linode',
+          service_type: 'linode',
+        },
+        {
+          label: 'Databases',
+          service_type: 'dbaas',
+        },
+      ],
     };
 
     return HttpResponse.json(response);
   }),
-  http.get('*/monitor/services/:serviceType/dashboards', () => {
+  http.get('*/monitor/services/:serviceType/dashboards', ({ params }) => {
     const response = {
-      data: [
+      data: [] as any[],
```

Comment:
```suggestion
      data: [] as Dashboard[],
```
use this interface instead of any
https://github.com/nikhagra-akamai/manager/blob/0bc9d73c4f351b6718061891cc6a9036ca4dff89/packages/api-v4/src/cloudpulse/types.ts#L1

Reply:
Updated